packages/manager/modules/pci/src/projects/project/storages/databases/translations/Messages_de_DE.json

```diff
@@ -29,5 +29,7 @@
   "pci_database_node_type_memory": "{{memorySize}} RAM",
   "pci_database_node_type_storage": "{{storageSize}} SSD Storage",
   "pci_database_node_type_storage_range": "{{storageMinSize}} bis {{storageMaxSize}} SSD Storage",
-  "pci_database_node_type_label": "Node-Modell"
-}
\ No newline at end of file
+  "pci_database_node_type_label": "Node-Modell",
+  "pci_database_footer_part_1": "Alle Produktnamen, Logos und Marken sind Besitz der jeweiligen Eigentümer. Alle auf dieser Website verwendeten Namen von Unternehmen, Produkten und Dienstleistungen dienen ausschließlich der Identifizierung. Die Verwendung dieser Namen, Logos und Marken bedeutet keine Genehmigung oder Bestätigung.",
+  "pci_database_footer_part_2": "*Redis ist eine Marke von Redis Ltd. Alle damit verbundenen Rechte sind Redis Ltd vorbehalten. Jede Nutzung durch OVH SAS dient lediglich der Information. Es wird kein Sponsoring, keine Unterstützung oder Verbindung zwischen Redis und OVH SAS zum Ausdruck gebracht."
```

Comment:
"*Redis" is same for all languages. Its better to add it to the constant file and pass to translation string as a variable. Since translations are already done,  this can be taken later. NOT A BLOCKER FOR RELEASE.